olp-cpp-sdk-core/src/http/DefaultNetwork.cpp

```diff
@@ -78,5 +110,12 @@ void DefaultNetwork::AppendDefaultHeaders(Headers& request_headers) const {
                          default_headers_.end());
 }
 
+void DefaultNetwork::LockStatistics(
+    uint8_t bucket_id, std::function<void(Statistics&)>&& callback) {
+  buckets_.locked([bucket_id, &callback](BucketsContainer& container) {
```

Comment:
```suggestion
  buckets_.locked([&](BucketsContainer& container) {
```
Both can be accessed by reference.